compiler/ras/Debug.cpp

```diff
@@ -4949,8 +4949,8 @@ void TR_Debug::setupDebugger(void *addr)
          char * Argv[20];
 
          yield();
-         sprintf(cfname, "_%ld_", getpid());
-         sprintf(pp,"%ld", ppid);
+         sprintf(cfname, "_%ld_", (int64_t)getpid());
+         sprintf(pp, "%ld", (int64_t)ppid);
```

Comment:
I'm not sufficiently familiar with different hardware platform, OS, compiler combinations to know whether `int64_t` will always be equivalent to `long` - particularly for Windows x86-32.

Another suggestion that I've read about for formatting `pid_t` is to cast to `intmax_t` and use `%jd` for the format specifier.  That might be more safe.

Reply:
Assuming `int64_t` is reasonable (I expect it is), this should use `OMR_PRId64` (from `omrformatconsts.h`) instead of `ld`:
```c
         sprintf(cfname, "_%" OMR_PRId64 "_", (int64_t)getpid());
         sprintf(pp, "%" OMR_PRId64, (int64_t)ppid);
```

Reply:
I see how `%ld` isn't necessarily the best choice since `int64_t` may not always be `long`. Thank you for the suggestion of the safer format specifier, Keith!